modules/oe_content_organisation/config/install/core.entity_form_display.node.oe_organisation.default.yml

```
@@ -3,14 +3,16 @@ status: true
 dependencies:
   config:
     - field.field.node.oe_organisation.body
-    - field.field.node.oe_organisation.oe_organisation_overview
+    - field.field.node.oe_organisation.oe_organisation_org_chart
```

Let's simply call this `oe_organisation_chart`:
```suggestion
    - field.field.node.oe_organisation.oe_organisation_chart
```